lib/server.js

```diff
@@ -109,7 +110,15 @@ server.restartBrowser = function() {
 	this.browser.kill();
 };
 
-
+server.gracefulBrowserRestart = function() {
+	this.isBrowserConnected = false;
+	util.log(`Graceful restarting ${this.browser.name}`);
```

Comment:
Do we need to protect `this.browser` for undefined property lookups?

Reply:
This library requires `server.init()` to have been to called, to function at all. It definitely isn't the best API, but I'm not going to re-work a bunch of code to fix that right now.